### DISCUSSION TITLE
Mountain Duck loosing mounts

### DISCUSSION THREAD

**Discussion by iBobik** [iterate-ch/cyberduck#14526] [Q&A]
Hi,
I wanted to use MD as a replacement for official Google Drive nad Dropbox apps, so I configured smart sync for both accounts. It works well after app start, but after few hours (and maybe sleep, network change etc.) mounts stop working - folders are empty and only disconnect and connect fix it again.

So basically unusable for normal work.

**Comment by Cassie205** [comment]
Please send a [debug log](https://docs.cyberduck.io/mountainduck/support/#debug-log) and a screenshot for the described issue to support@mountainduck.io for further investigation on the issue.

**Comment by tatze96** [comment]
The mounted drive didn't show the files due to colons within the file names. The colon counts into the characters [reserved](https://docs.cyberduck.io/mountainduck/issues/#files-and-folders-not-synced-from-server) for operating system processes.
